servlet/4.0/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Jakarta Servlet 4.0"
+date: 2019-07-24T23:21:09+0200
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
+
+* [Jakarta Servlet 4.0 Specification Document](./servlet-4.0.pdf) (PDF)
+* [Jakarta Servlet 4.0 Specification Document](./servlet-4.0.html) (HTML)
+* [Jakarta Servlet 4.0 Javadoc](./apidocs)
+* Maven coordinates
+  * [jakarta.servlet:jakarta.servlet-api:jar:4.0.3](https://search.maven.org/artifact/jakarta.servlet/jakarta.servlet-api/4.0.3/jar)
+
+# Ballots
+
+## Release Review
+
+
+|                       |  Yes    | No      | Abstain  |
+|-----------------------|---------|---------|----------|
+|Fujitsu                |         |         |          |
+|IBM                    |         |         |          |
+|Oracle                 |         |         |          |
+|Payara                 |         |         |          |
+|Red Hat                |         |         |          |
+|Tomitribe              |         |         |          |
+|EE4J PMC               |         |         |          |
+|Participant Members    |         |         |          |
+|Committer Members      |         |         |          |
+
+
+# Compatible Implementations
+
+* GlassFish 5.1
```

Comment:
Need a link to the proper CI binaries.

Reply:
You can use 

[Eclipse GlassFish 5.1.0](https://www.eclipse.org/downloads/download.php?file=/glassfish/glassfish-5.1.0.zip)
--

Reply:
For the Servlet TCK, you can use
[eclipse-servlet-tck-4.0.0.zip](http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-servlet-tck-4.0.0.zip)